tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
 	"compilerOptions": {
-		"target": "es5",
+		"target": "esnext",
```

Comment:
I would prefer putting some concrete ES version here, but as I can see it's the same in the Vue client so if you want to keep it consistent across the project repos or you have some other reason for that, it's fine for me.